src/pages/questions/ui/QuestionsPage.module.scss

```
@@ -0,0 +1,55 @@
+@import 'src/app/styles/variables/global.scss';
+
+.QuestionsPage {
+    padding: 16px 0 24px 0;
+    flex: 1;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    gap: 28px;
+}
+
+.title {
+    font-size: 17px;
+    font-weight: 400;
+    line-height: 22px;
+    text-align: left;
+    color: var(--color-primary-200)
+}
+
+.questionsBlock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 28px;
+    max-width: 520px;
+    z-index: 1;
+}
```

```suggestion
.questionsBlock {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    gap: 28px;
    z-index: 1;
}
```

мне кажется, тут не нужен max-width